package.json

```diff
@@ -338,7 +338,7 @@
     "semantic-release": "24.2.3",
     "tar": "7.4.3",
     "tmp-promise": "3.0.3",
-    "ts-node": "10.9.2",
+    "tsx": "4.19.3",
```

Comment:
Now that we are having Node 22.13.0 as minimum version we could also use the experimental flag to remove types and drop this 3rd party dep. 🤔

Reply:
it's not yet in main and tsx solves some import inconsistencies